scikit_mol/conversions.py

```diff
@@ -13,6 +13,13 @@ def __init__(self, parallel: Union[bool, int] = False):
         self.parallel = parallel
         self.start_method = None  #TODO implement handling of start_method
 
+    def get_feature_names_out(self, input_features=None):
```

Comment:
Is this needed by the base sklearn classes? I can't seem to understand where it's used otherwise

Reply:
This is what makes the "pandas output" mechanism work. If this method is defined, then the base classes will use it to generate the column names, and return the result as a Pandas DataFrame. If this method is not defined, we get an error when trying to set the pandas output for transformers.

Reply:
OK, thanks for the clarification